pkg/kubelet/kubelet_test.go

```diff
@@ -2500,6 +2617,12 @@ func TestPurgingObsoleteStatusMapEntries(t *testing.T) {
 	testKubelet.fakeCadvisor.On("MachineInfo").Return(&cadvisorapi.MachineInfo{}, nil)
 	testKubelet.fakeCadvisor.On("DockerImagesFsInfo").Return(cadvisorapiv2.FsInfo{}, nil)
 	testKubelet.fakeCadvisor.On("RootFsInfo").Return(cadvisorapiv2.FsInfo{}, nil)
+	versionInfo := &cadvisorapi.VersionInfo{
```

Comment:
Does the VersionInfo need to be set required for the test to pass? I am only asking because kubemark relies on the cadvisor fake (https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/cadvisor/testing/cadvisor_fake.go) which returns &cadvisorapi.VersionInfo{}. Just wanted to make sure that this won't break kubemark.